dozer-ingestion/mysql/src/binlog.rs

```diff
@@ -434,14 +433,38 @@ impl BinlogIngestor<'_, '_, '_> {
                         }
 
                         let table = table_cache.get_table_details(table_index).unwrap();
-                        self.handle_rows_event(&rows_event, &table, tme).await?;
+                        let mut pos = BinlogPosition {
+                            filename: self.next_position.filename.clone(),
+                            seq_no: seq_in_transaction,
+                            position: transaction_position,
+                        };
+
+                        pos.seq_no = self
```

Comment:
`seq_in_transaction` is no longer updated.

Reply:
Thanks, removed it.